programs/account-compression/src/instructions/initialize_address_merkle_tree.rs

```diff
@@ -42,6 +44,14 @@ pub fn process_initialize_address_merkle_tree<'info>(
                 .map_err(|_| AccountCompressionErrorCode::IntegerOverflow)?,
         )
         .map_err(ProgramError::from)?;
+    let address_merkle_tree_inited = address_merkle_tree.load_merkle_tree_mut()?;
 
+    // Initialize the address merkle tree with the bn254 Fr field size - 1
+    // This is the highest value that you can poseidon hash with poseidon syscalls.
+    // Initializing the indexed Merkle tree enables non-inclusion proofs without handling the first case specifically.
+    // However, it does reduce the available address space by 1.
+    let init_value = BigUint::from_str_radix(FIELD_SIZE_SUB_ONE, 10).unwrap();
```

Comment:
And we could also move that to `init()` in `merkle-tree/indexed/src/lib.rs`. We already define `0` there.